src/runtime-rs/crates/resource/src/network/utils/link/mod.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+mod create;
+pub use create::{create_link, LinkType};
+mod driver_info;
+pub use driver_info::{get_driver_info, DriverInfo};
+mod manager;
+pub use manager::get_link_from_message;
+
+use std::os::unix::io::RawFd;
+
+use netlink_packet_route::link::nlas::State;
+
+#[macro_export]
```

Comment:
I think we can remove the #[macro_export] because the macro is not used cross crates.

Reply:
Thanks for your review, remove the `#[macro_export] ` and move macro to `macros.rs`